modules/monitoring/manifests/checks/smokey.pp

```diff
@@ -21,7 +21,10 @@
 
   include govuk::apps::smokey
 
-  # TODO: Should this really run as root?
+  govuk_user { 'smokey':
+    fullname    => 'Smokey',
+  }
```

Comment:
I think this class is more intended for a person, not a service (in the admin, deploy and adm groups by default). Something like this would be more appropriate (and you might be able to omit the shell):
```
  user { 'smokey':
    ensure => present,
    shell  => '/bin/bash',
  }
```

Reply:
When I did this in staging, I also made it a `system` user, as that seemed sortof sensible.

Reply:
https://github.com/alphagov/govuk-puppet/pull/7817